spec/controllers/coronavirus/sub_sections_controller_spec.rb

```diff
@@ -144,6 +144,17 @@
       expect(sub_section.title).to eq(title)
       expect(sub_section.content).to eq(content)
     end
+
+    context "given invalid content" do
+      let(:content) { "invalid content" }
+
+      it "doesn't change the subsection" do
+        subject
+        sub_section.reload
```

Comment:
These two lines aren't needed (well the first one `subject` actually renders the assertion on line 154 a non-check as subject is memo'd)

Reply:
ah nice one, dangers of copying and pasting things!